src/adera/activescene/VehicleBuilder.cpp

```diff
@@ -201,15 +201,15 @@ VehicleData VehicleBuilder::finalize_release()
 
         // reserve node-to-machine partitions
         rPerNodeType.nodeToMach.data_reserve(rPerNodeType.m_connectCountTotal);
-        for (NodeId node : rPerNodeType.nodeIds.bitview().zeros())
+        for (NodeId node : rPerNodeType.nodeIds)
```

Comment:
I don't disagree with this change, but I'm not really sure how related to the Jolt physics change it is.

Maybe this should be in it's own stand-alone PR?

Reply:
It is ! Sorry it is not the cleanest, I have two other PRs open and this is one of them. I kinda figured I would start with the most advanced version of the code I could, as this PR will take me quite some time anyway.